addons/mass_mailing/static/src/js/mass_mailing_widget.js

```diff
@@ -84,6 +85,7 @@ var MassMailingFieldHtml = FieldHtml.extend({
                 changes: _.object([fieldName], [self._unWrap($editable.html())])
             });
 
+            $editable.html(self.value);
```

Comment:
Seems a bit suspicious to need to introduce that after 740168ce8d27da3d6a7156d2d79655a898394923 removed 

    self.wysiwyg.setValue(result.html);

at this exact same place.

Reply:
The commit you mention is the one that introduced the bug. That line was removed and shouldn't have been. I could have used setValue instead since what it does is exactly what my line does. But I felt like that was an unnecessary indirection.
I believe the person who removed it (@Goaman ?) probably did so because this line without the `self._doAction();` above caused a more visible issue and they were in a hurry ^^'
`result` doesn't exist anymore and `result.html` was in fact the editor's html pre-inline-modifications, exactly like my `self.value` is here.
![image](https://user-images.githubusercontent.com/24205914/131858986-b448b525-16d0-4d4e-8f6c-12133182283c.png)

Reply:
> That line was removed and shouldn't have been. I could have used setValue instead since what it does is exactly what my line does. But I felt like that was an unnecessary indirection.

Seems like a good way to introduce bugs in the future if things are hooked on `setValue`. Plus `setValue` wouldn't be sec-flagged.

Reply:
> Seems like a good way to introduce bugs in the future if checks or transformations get hooked on `setValue`.

You're right, I'll change it.

> Plus setValue wouldn't be sec-flagged.

Not triggering the security check while actually doing a `.html()` felt like cheating I guess ^^ But seriously that could have been kind of dangerous.

Reply:
> Not triggering the security check while actually doing a .html() felt like cheating I guess ^^ But seriously that could have been kind of dangerous.

That's true but at the same time if it's an issue with the widget it's the widget which should be fixed.

Reply:
Wait actually no I shouldn't change it. I precisely don't want any side effects here so if some are introduced later that could spell disaster. I just want to set the editable node's html, no magic.